### ISSUE TITLE
--pull output appears in --test-report

### ISSUE THREAD

**Issue by reschenburgIDBS** [GoogleContainerTools/container-structure-test#245]
Hi

when running cst with --pull and --test-report, the --pull output appears in the --test-report file, which is annoying as I then can't parse the test report as json without going through the extra step of extracting the json first.
Sadly, when providing -q, the test-report is empty.. which is counter intuitive as -q should really only quiet stdout.

It would be good to have test-report always generate the test-report only, regardless of other parameters.

**Comment by nkubala** [comment]
@reschenburgIDBS yeah this would be good to update, I agree the output could use some love. are you interested in sending a PR? our team isn't actively working on this project at the moment, but I'm happy to review any contributions.
